package.json

```diff
@@ -2,55 +2,51 @@
   "name": "@hoodie/task-client",
   "description": "client api for asynchronous task queue, using PouchDB for sync",
   "main": "index.js",
-  "publishConfig": {
-    "access": "public"
-  },
   "scripts": {
     "prebuild": "rimraf dist && mkdirp dist",
-    "build": "browserify index.js --standalone=TaskQueue > dist/hoodie-task-client.js",
-    "postbuild": "uglifyjs dist/hoodie-task-client.js -mc > dist/hoodie-task-client.min.js",
+    "build": "browserify -s Task index.js > dist/hoodie-task-client.js",
     "pretest": "standard",
-    "test": "npm run -s test:node | tap-spec",
-    "test:node": "node tests",
-    "test:watch": "gaze 'clear && node tests | tap-min' 'tests/**/*.js' 'index.js' 'lib/**/*.js'",
+    "test": "standard",
     "test:coverage": "istanbul cover tests",
     "test:coverage:upload": "istanbul-coveralls",
-    "semantic-release": "semantic-release pre && npm publish && semantic-release post"
+    "test:node": "node tests",
+    "semantic-release": "semantic-release pre && npm publish && semantic-release post",
+    "postpublish": "greenkeeper-postpublish"
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/hoodiehq/hoodie-task-client.git"
+    "url": "git+https://github.com/hoodiehq/hoodie-task-client.git"
```

Comment:
I think it was residual from my clone of the `store-client` module (what I used as a basis for the refactor).  Fixed!  Thanks!